ui/component/user-settings-list.component.ts

```diff
@@ -0,0 +1,12 @@
+import { RowSpaced } from "../core/html.elements"
+import { Component } from "../core/html.interfaces"
+import { UserModel } from "../page/user.page"
+import { UserSettingComponent } from "./user-setting.component"
+
+export const UserSettingsList: Component<UserModel> = (user: UserModel) => {
+
+    return RowSpaced(
+        ...user.settings.getSettings().map(setting => UserSettingComponent({user: user.user, userSettings: setting}))
```

Comment:
Since this is just few lines of code, I'd say we don't need it as a separate component.

Reply:
Resolved in https://github.com/mbrizic/ratethegame/pull/28/commits/c56ac3d31ffb3beebc213748977bf5e2eb22195e